vllm/engine/metrics.py

```diff
@@ -59,7 +59,19 @@ def __init__(self, labelnames: List[str], max_model_len: int):
             name="vllm:cpu_cache_usage_perc",
             documentation="CPU KV-cache usage. 1 means 100 percent usage.",
             labelnames=labelnames)
-
+        #   Speculative infer Status in %
```

Comment:
Please add better descriptions.

Please name these:
- `vllm:spec_decode_system_efficiency`
- `vllm:spec_decode_boost_ratio`
- `vllm:spec_decode_draft_acceptance_rate`

What is the difference between these?

Reply:
We prefer to use Counters >> Gauges

Is there a way these metrics could be expressed as Counters with the `rate` function used in PromQL to compute the rates?

Reply:
maybe we could directly express the total emitted token, along with steps number? so that user could do the cal they want with those counters?